StackTraceLogEvent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Serilog.Events;
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+
+namespace Sharpbrake.Serilog
+{
+    internal class StackTraceLogEvent : LogEvent
+    {
+        public StackTrace StackTrace { get; set; }
+
+        public StackTraceLogEvent(LogEvent logEvent)
+            : base(
+                logEvent.Timestamp,
+                logEvent.Level,
+                logEvent.Exception,
+                logEvent.MessageTemplate,
+                logEvent.Properties.Select(x => new LogEventProperty(x.Key, x.Value))
+                )
+        {
+            StackTrace = GetStackTrace();
+        }
+
+        private StackTrace GetStackTrace()
+        {
+#if !NETSTANDARD1_4
```

Comment:
why do you need this? we are targeting netstandard2 and 2.1

Reply:
fixed